src/hooks/logIn.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import axios from 'axios'
+
+interface userType{
```

Comment:
we know that it is a type, please rename it to "User" and move it to general types file